samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/EventLogBean.java

```diff
@@ -176,5 +159,25 @@ public boolean getEnabledIpAddress(){
    public void setEnabledIpAddress(boolean enabledIpAddress){
 	   this.enabledIpAddress = enabledIpAddress;
    }	
+
+   public String getDataTableConfigJson() {
+        Gson gson = new GsonBuilder().create();
+        JsonElement jsonElement = gson.toJsonTree(this.dataTableConfig);
+        JsonObject jsonObject = (JsonObject) jsonElement;
+
+        return jsonObject.toString();
+   }
+
+   public String getExportUrl() {
+       log.info("filteredAssessmentId {}", filteredAssessmentId);
+        StringBuilder url = new StringBuilder("/samigo-app/servlet/ExportEventLog");
+        url.append("?" + ExportEventLogServlet.PARAM_SITE_ID + "=" + siteId);
```

Comment:
Umm, maybe something like this is cleaner?

    URIBuilder uriBuilder = new URIBuilder("/samigo-app/servlet/ExportEventLog");
    uriBuilder.addParameter(ExportEventLogServlet.PARAM_SITE_ID, siteId);
    if (filteredAssessmentId != null && filteredAssessmentId != -1L) {
        uriBuilder.addParameter(ExportEventLogServlet.PARAM_ASSESSMENT_ID, filteredAssessmentId);
    }